src/components/studies/scheduler/IntroInfo.tsx

```diff
@@ -93,39 +112,59 @@ const IntroInfo: React.FunctionComponent<IntroInfoProps> = ({
         }
         className={classes.formControl}
         labelPlacement="start"
-        control={<SimpleTextInput fullWidth onChange={e=> setStudyName(e.target.value)} style={{marginBottom: 0}}/>}
-      />
-      <FormControlLabel
-        classes={{labelPlacementStart: classes.labelDuration  }}
-        style={{ /*justifyContent: 'space-between'*/ }}
-        label={
-          <Box width="210px" marginRight="40px">
-            <strong className={classes.headerText}>
-              How long will the study run for?
-            </strong>
-            <br /> <br />
-            <div className={classes.description}>
-              This is the date the study will permanently close. Make sure to
-              add any extra buffer days you might need should the study go
-              longer.
-            </div>{' '}
-          </Box>
-        }
-        className={classes.formControl}
-        labelPlacement="start"
         control={
-          <Duration
-            onChange={e => setDuration(e.target.value)}
-            durationString={duration || ''}
-            unitLabel="study duration unit"
-            numberLabel="study duration number"
-            unitData={DWsEnum}
-            isIntro={true}
-          ></Duration>
+          <SimpleTextInput
+            fullWidth
+            onChange={e => setStudyName(e.target.value)}
+            style={{ marginBottom: 0 }}
+          />
         }
       />
-      <Divider className={classes.divider}></Divider>
+      <Divider className={classes.divider1}></Divider>
+      <Box className={classes.middleContainer}>
+        <FormControlLabel
+          classes={{ labelPlacementStart: classes.labelDuration }}
+          style={
+            {
+              /*justifyContent: 'space-between'*/
+            }
+          }
+          label={
+            <Box width="210px" marginRight="40px">
+              <strong className={classes.headerText}>
+                How long will the study run for?
+              </strong>
+              <br /> <br />
+              <div className={classes.description}>
+                This is the date the study will permanently close. Make sure to
+                add any extra buffer days you might need should the study go
+                longer.
+              </div>{' '}
+            </Box>
+          }
+          className={classes.formControl}
+          labelPlacement="start"
+          control={
+            <Duration
+              onChange={e => setDuration(e.target.value)}
+              durationString={duration || ''}
+              unitLabel="study duration unit"
+              numberLabel="study duration number"
+              unitData={DWsEnum}
+              isIntro={true}
+            ></Duration>
+          }
+        />
+        <p className={classes.weekInformation}>
+          1 year = 52 weeks
+          <br></br>2 year = 104 weeks
```

Comment:
instead of using  double <br> let's make it a <ul> and you can use the weekInformation class to style the li's underneath